modules/meteroid/src/api_rest/plans/mapping.rs

```diff
@@ -0,0 +1,17 @@
+use crate::api_rest::plans::model::Plan;
+use meteroid_store::domain;
+
+pub fn domain_to_rest(d: domain::PlanOverview) -> Plan {
+    Plan {
+        local_id: d.local_id,
```

Comment:
the name we expose to the outside world should be just 'id'. local_id is our internal name, there is a chance we will get rid of uuids and use just a single id.